.taskcluster.yml

```diff
@@ -207,7 +207,6 @@ tasks:
                               git fetch ${repository} --tags
                               && git config advice.detachedHead false
                               && git checkout ${tag}
-                              && pip install pyyaml
```

Comment:
I assume that we need the new Docker image on Docker Hub (and update the version here) before we can land this part -- or why pyyaml never needed here?. However to re-build the image it would be nice to land the Dockerfile changes first. Maybe we should separate those changes? :)

Reply:
You're right, `pyyaml` is needed for that task, too.
It's true that there's a circular dependency-ish thing going on here where it's weird to deploy the docker image before it's merged, but you can't merge until the new docker image is deployed.

I see below that you rebuilt the docker image, but next time I'll separate such changes :+1: